jobs/cloud_controller_ng/templates/ccng_monit_http_healthcheck.sh.erb

```diff
@@ -30,28 +30,24 @@ PORT=<%= p("cc.public_tls.port") %>
 PROTOCOL="https"
 URL="https://${HOST}:${PORT}/healthz"
 
-source /var/vcap/packages/capi_utils/monit_utils.sh
-
-echo $(date --rfc-3339=ns) 'Waiting for Cloud Controller to initially become healthy at'
-
-wait_for_server_to_become_healthy "${URL}" "<%= p("cc.api_post_start_healthcheck_timeout_in_seconds") %>"
-
-echo $(date --rfc-3339=ns) 'Initial check passed, will now restart CC over on repeated failures'
+echo $(date --rfc-3339=ns) 'Will restart CC over on repeated failures'
 
 trap log_failure EXIT
 
 # if we fail to curl it 5 times in a row across 50 seconds, die so monit will restart us
```

Comment:
Can you update the comment to reflect configurable value? 
Thanks, 
Teal & @sethboyles

Reply:
Yes, that makes sense.